<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="navGroupBox">
        <property name="title">
         <string>Items</string>
        </property>
        <widget class="QSplitter" name="splitter">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>24</y>
           <width>61</width>
           <height>271</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QPushButton" name="monitorButton">
          <property name="text">
           <string>Monitor</string>
          </property>
         </widget>
         <widget class="QPushButton" name="programButton">
          <property name="text">
           <string>Program</string>
          </property>
         </widget>
         <widget class="QPushButton" name="auxButton">
          <property name="text">
           <string>Aux Data</string>
          </property>
         </widget>
         <widget class="QPushButton" name="helpButton">
          <property name="text">
           <string>Help</string>
          </property>
         </widget>
        </widget>
        <zorder>splitter</zorder>
        <zorder>stackedWidget</zorder>
        <zorder>stackedWidget</zorder>
        <zorder>stackedWidget</zorder>
        <zorder>stackedWidget</zorder>
        <zorder>stackedWidget</zorder>
        <zorder>stackedWidget</zorder>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <widget class="QWidget" name="monitorPage">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="labelMonitor">
            <property name="text">
             <string>&lt;h3&gt;Monitor&lt;/h3&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSplitter" name="splitter_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="labelH1">
             <property name="geometry">
              <rect>
               <x>150</x>
               <y>10</y>
               <width>20</width>
               <height>25</height>
              </rect>
             </property>
             <property name="text">
              <string>&lt;h4&gt;H1&lt;/h4&gt;</string>
             </property>
            </widget>
            <widget class="QProgressBar" name="t1ProgressBar">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>60</y>
               <width>81</width>
               <height>21</height>
              </rect>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
            <widget class="QProgressBar" name="h2ProgressBar">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>60</y>
               <width>121</width>
               <height>25</height>
              </rect>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
            <widget class="QLabel" name="labelH2">
             <property name="geometry">
              <rect>
               <x>150</x>
               <y>60</y>
               <width>20</width>
               <height>25</height>
              </rect>
             </property>
             <property name="text">
              <string>&lt;h4&gt;H1&lt;/h4&gt;</string>
             </property>
            </widget>
            <widget class="QProgressBar" name="t2ProgressBar">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>10</y>
               <width>121</width>
               <height>25</height>
              </rect>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
            <widget class="QLabel" name="labelT2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>60</y>
               <width>33</width>
               <height>25</height>
              </rect>
             </property>
             <property name="text">
              <string>&lt;h4&gt;T2&lt;/h4&gt;</string>
             </property>
            </widget>
            <widget class="QProgressBar" name="h1ProgressBar">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>10</y>
               <width>89</width>
               <height>25</height>
              </rect>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
            <widget class="QLabel" name="labelT1">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>31</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>&lt;h4&gt;T1&lt;/h4&gt;</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QSplitter" name="splitter_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSplitter" name="splitter_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSplitter" name="splitter_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="programPage">
         <widget class="QLabel" name="labelProgram">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>0</y>
            <width>111</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;h3&gt;program Set&lt;/h3&gt;</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="auxPage">
         <widget class="QLabel" name="labelAux">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>0</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;h3&gt;Aux Data&lt;/h3&gt;</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="helpPage">
         <widget class="QLabel" name="labelHelp">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>0</y>
            <width>41</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;h3&gt;Help&lt;/h3&gt;</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>608</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
